cmd/empire/main.go

```diff
@@ -66,6 +69,12 @@ var Commands = []cli.Command{
 				Usage: "Whether to run the migrations at startup or not",
 			},
 			cli.StringFlag{
+				Name:   FlagEventsBackend,
+				Value:  "",
+				Usage:  "The backend implementation to use to send event notifactions",
```

Comment:
Is there an easy way with codegangsta's CLI to give a list of choices for an option and display that with the help?